samples/cpp/kalman.cpp

```diff
@@ -35,7 +35,7 @@ int main(int, char**)
     Mat state(2, 1, CV_32F); /* (phi, delta_phi) */
     Mat processNoise(2, 1, CV_32F);
     Mat measurement = Mat::zeros(1, 1, CV_32F);
-    char code = (char)-1;
+    char code = 0;
```

Comment:
If we leave it unitialized, then OpenCV generates a warning of an uninitialized char, which is seen as an error due to the `threat warnings as errors` compiler flag.